src/sidebar/components/Annotation/AnnotationHeader.js

```diff
@@ -133,12 +133,9 @@ function AnnotationHeader({
           displayName={authorDisplayName}
         />
         {showReplyButton && (
-          <Button
-            className="AnnotationHeader__reply-toggle"
```

Comment:
Can you also remove the css rule in the sass file?

```
  &__reply-toggle {
    @include buttons.button--link;
    padding: 0 var.$layout-space--xsmall;
  }
  ```

Reply:
Sure thing!